.travis.yml

```diff
@@ -162,8 +162,6 @@ jobs:
           "
 
   allow_failures:
-    - name: "Go on ARM"
-    - name: "Go on s390x"
```

Comment:
Depending on the build's stability we may need to allow failures for s390x again.